src/common/ast-path.js

```diff
@@ -48,6 +73,23 @@ class AstPath {
     return getNodeHelper(this, count + 1);
   }
 
+  #getSiblingNode(offset) {
+    const { stack } = this;
+    const { length } = stack;
+    if (length < 3) {
+      return null;
+    }
+    const number = stack[length - 2];
+    if (typeof number !== "number") {
+      return null;
```

Comment:
Should we throw errors, when getting siblings if it's not in array?

Reply:
Let's add assertions.

Reply:
After all, `map` and `each` don't check `isArray`.